Disable Python embedding and remove build-machine flags from public version metadata.
Recipe PR: https://github.com/JuliaPackaging/Yggdrasil/pull/14202 (related core-LAL packaging proposal)

--- a/lalsimulation/configure.ac
+++ b/lalsimulation/configure.ac
@@ -147,7 +147,7 @@
 AC_CHECK_HEADERS([gsl/gsl_errno.h],,[AC_MSG_ERROR([could not find the gsl/gsl_errno.h header])])
 
 # check for python3-embed
-PKG_CHECK_MODULES([PYTHON3_EMBED],[python-${PYTHON_VERSION}-embed],[have_python3_embed=true],[have_python3_embed=false])
+have_python3_embed=false
 if test "x$have_python3_embed" = xtrue; then
 
     # separate out -l and -L LIBS
--- a/gnuscripts/lalsuite_vcs_info.am
+++ b/gnuscripts/lalsuite_vcs_info.am
@@ -15,6 +15,6 @@
 $(vcs_build_info_source): $(vcs_info_sources) $(build_info_source)
 $(build_info_source): Makefile $(build_info_dependencies)
 	$(AM_V_GEN)echo '/* $@: generated by Makefile */' >$@; \
-	echo '#define $(PACKAGE_NAME_UCASE)_CONFIGURE_ARGS "'"$(ac_configure_args)"'"' >>$@; \
+	echo '#define $(PACKAGE_NAME_UCASE)_CONFIGURE_ARGS "'"BinaryBuilder: shared libraries; Python and OpenMP disabled; HDF5 enabled"'"' >>$@; \
 	echo '#define $(PACKAGE_NAME_UCASE)_CONFIGURE_DATE "'"$(configure_date)"'"' >>$@; \
 	echo '#define $(PACKAGE_NAME_UCASE)_BUILD_DATE "'`$(top_srcdir)/gnuscripts/reproducible-date +"%Y-%m-%dT%H:%M:%S%z"`'"' >>$@
